### PR TITLE
added support for csv and xlsx input

### LINKED ISSUE #14: Enable CSV / Excel input file
The current input file is using simple txt. Some people may have a CSV or Excel file, so we may need to allow them to directly use it as an input.

### PR DESCRIPTION
**Pull request by pradeepb2505**
Closes #14 
Example command - `pipenv run python3 main.py -i input.csv   `

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -11,3 +11,5 @@ python_version = ">=3.5"
 [packages]
 python-dotenv = "*"
 requests = "*"
+pandas = "*"
+openpyxl = "*"
```
Comment (rizqirizqi) [inline Pipfile:15]: we can remove openpyxl if it's unused

Comment (pradeepb2505) [reply Pipfile:15]: This is needed for reading the .xlsx files

Comment (rizqirizqi) [reply Pipfile:15]: Ah I see, okay nice :+1:

**Review by rizqirizqi — APPROVED** [review]

**Comment by rizqirizqi** [comment]
Thanks for the PR! 💚

**Pull request merged**